packages/vsce/__tests__/__e2e__/01_EditProjectTeamConfiguration.test.ts

```diff
@@ -86,6 +86,19 @@ describe("Create Project Level Team Configuration File Scenario", () => {
     // Check zowe.config.json was opened - could check content here
     expect(titles.some((title) => title.startsWith("zowe.config.json"))).is.true;
 
+    // Open the editor for zowe.config.json
+    const editor = await editorView.openEditor("zowe.config.json") as TextEditor;
+   
+    const isWmAvailable = await editor.getText()
+    if(!isWmAvailable.includes("wiremock_server")){
+      const wiremockConfig = ",\n\"wiremock_server\": {\n   \"type\": \"cics\",\n\"properties\": {\n     \"host\": \"localhost\",\n\"port\": 8080,\n\"rejectUnauthorized\": false,\n\"protocol\": \"http\"\n}\n}"
+    
+      // add the wiremock configuration at the given coordinates
+      await editor.typeTextAt(47,10,wiremockConfig);
+      await editor.save();
+      const editorText = await editor.getText();
+      expect(editorText).to.contain("wiremock_server");
+    }
```

Comment:
I really like how you've looked into how to do this, and this will definitely come in useful!

However, because you've added the `wiremock_server` profile to the zowe.config.json we are using in this test, this will never executed right? I also believe it's good practice to know the environment the tests are running in (knowing what profiles are present so we can accurately test them) so I'm not sure we need this uncertainty. 

Perhaps an alternative is tweaking this to open the editor and reading the content still, but changing it from adding the profile if it's not there, to expecting it to be there. Something like...

```suggestion
    expect(isWmAvailable.includes("wiremock_server")).to.be.true;
```

Reply:
i was wondering about this too - i think the intention is to add the wiremock_server info into the the file at runtime ... but specifying the exact point in the file is brittle - likely to break if we update the format of that file at all.

i'm not sure exactly what the solution is...